### ISSUE TITLE
Publish not working in Reunion 0.8

### ISSUE THREAD

**Issue by huoyaoyuan** [microsoft/microsoft-ui-xaml#5480]
Visual Studio version: 16.10.3

Just invoke publish-create package in Visual Studio.

Error message:
```
Can't find asset file “....\obj\wappublish\win10-x86\project.assets.json”
```

There is a `win-x86` folder there, instead of `win10-x86`. The project uses `<RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>` correctly.

**Comment by StephenLPeters** [comment]
@alwu-msft FYI

**Comment by alwu-msft** [comment]
Duplicate of #5248. This is expected to fixed by VS 16.10.4.
